MaxText/layers/attentions.py

```diff
@@ -1208,6 +1212,7 @@ class Attention(nn.Module):
   ragged_block_size: int = 256
   use_qk_norm: bool = False
   query_pre_attn_scalar: float | None = None
+  use_bias_in_projection: bool = False
```

Comment:
nit: `use_bias_in_projections` (plural) and please add comment:
looks like it's being applied to many projections (q, k, v) and output projections.

Reply:
fixed